manifests/install.pp

```diff
@@ -2,21 +2,24 @@
 #
 # @api private
 class qpid::install {
-  include qpid::tools
+
+  if $qpid::ensure == 'absent' {
+    $_package_ensure = 'purged'
+  } else {
+    $_package_ensure = $qpid::version
+  }
 
   package { $qpid::server_packages:
-    ensure => $qpid::version,
-    before => Class['qpid::tools'],
```

Comment:
This could in theory cause a regression of https://github.com/theforeman/puppet-qpid/pull/47 ?

Reply:
Good question, I'll add a test to see.